data/hai missions.txt

```diff
@@ -877,7 +877,7 @@ mission "Language: Wanderer 2B attacked"
 			branch with
 				has "Language: Wanderer 2B: active"
 
-			`The ship carrying Eruk, the Kach'cha takes off immediately before you. Coming out of the planets atmosphere you see it getting ready to jump, you also see the Redacted coming up fast behind it. At close range it opens fire with weapons much more powerful then a ship it size has any right to possess, the Kach'cha is disabled almost immediately. The "Redacted" then does something you have never seen any ship do before, it proceeds to ram the Kach'cha, utterly destroying both ships in the process.`
+			`The ship carrying Eruk, the Kach'cha takes off immediately before you. Coming out of the planets atmosphere you see it getting ready to jump, you also see the Redacted coming up fast behind it. At close range it opens fire with weapons much more powerful then a ship it size has any right to possess, the Kach'cha is disabled almost immediately. The Redacted then does something you have never seen any ship do before, it proceeds to ram the Kach'cha, utterly destroying both ships in the process.`
```

Comment:
Should probably replace the comma with a colon (, - :) after the phrase "The Redacted then does something you have never seen any ship do before,"